<launch>

  <!--<arg name="mode" value="none" />-->
  <!--<arg name="mode" value="fixed" />-->
  <!--<arg name="mode" value="adhoc" />-->
  <!--<arg name="mode" value="only2d" />-->
  <!--<arg name="mode" value="projection_full" />-->
  <arg name="mode" value="only3d" />
  <!--<arg name="mode" value="unfrozen" />-->
  <!--<arg name="mode" value="noattention" />-->

  <node pkg="nbv_3d_cnn_real_image" type="cnn_real_image_eval_gain" name="cnn_real_image_eval_gain" clear_params="true" output="screen">
    <param name="mode" type="string" value="$(arg mode)" />
    
    <param name="scenario_first_index" type="int" value="11" />
    <param name="scenario_last_index" type="int" value="15" />
    
    <param name="save_images" type="bool" value="true" />
    
    <param name="scenario_file_prefix" type="string" value="$(find nbv_3d_cnn_real_image)/data/training_images/scenario_" />
    <param name="gt_file_prefix" type="string" value="/virtual_images/" />
    <param name="environment_file_prefix" type="string" value="/partial_environments/" />
    
    <param name="mask_file_name" type="string" value="$(find nbv_3d_cnn_real_image)/data/training_images/scenario_1/virtual_images/mask.png" />
    
    <param name="evaluation_file_prefix" type="string" value="$(find nbv_3d_cnn_real_image)/data/evaluation/$(arg mode)/" />
    <param name="gt_evaluation_file_prefix" type="string" value="$(find nbv_3d_cnn_real_image)/data/evaluation/gt/" />
    
    <param name="eval_gain_file_prefix" type="string" value="$(find nbv_3d_cnn_real_image)/data/eval_gain/$(arg mode)/" />
  </node>

</launch>
